core/trie/trie_test.go

```diff
@@ -87,52 +88,296 @@ func TestFindCommonPath(t *testing.T) {
 }
 
 func TestTriePut(t *testing.T) {
+	t.Run("put to empty tree", func(t *testing.T) {
+		RunOnTempTrie(251, func(trie *Trie) error {
+			keyNum, _ := strconv.ParseUint("1101", 2, 64)
```

Comment:
Please don't ignore errors:

```suggestion
			keyNum, err := strconv.ParseUint("1101", 2, 64)
			require.NoError(t, err)
```
`require` is an accompanying package to `assert` from the same `testify` repo. You should use `require` where you don't expect an error and if it does happen you want to stop the test from running. `assert` will log the error but will still continue to execute the rest of the test. 

In this case, if `ParseUint` fails we don't want to continue, therefore, we should use `require` instead of `assert`.

As a general rule of thumb, use `require` for unexpected errors and use `assert` to define the pass/failing conditions of the test.